examples/website/3d-heatmap/app.js

```diff
@@ -11,6 +14,32 @@ const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
 const DATA_URL =
   'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv'; // eslint-disable-line
 
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 1.0
+});
+
+const pointLight1 = new PointLight({
+  color: [255, 255, 255],
+  intensity: 0.8,
+  position: [-0.144528, 49.739968, 80000]
+});
+
+const pointLight2 = new PointLight({
+  color: [255, 255, 255],
+  intensity: 0.8,
+  position: [-3.807751, 54.104682, 8000]
+});
+
+const lightingEffect = new LightingEffect({ambientLight, pointLight1, pointLight2});
```

Comment:
Is this correct? Doesn't the effect take an array of `pointLights`? or even `lights` with the latest luma changes?

Reply:
yeah, it is correct. This is the deck.gl user API which takes a collection (Object) of light sources, the API from luma.gl will be only used in deck.gl internally, won't exposed to deck.gl users.